components/AddForm.js

```diff
@@ -66,7 +65,7 @@ export default function AddForm() {
 const StyledFormContainer = styled.form`
   display: grid;
   gap: 20px;
-  margin: 30px 10px 10px 10px;
+  margin: 30px 10px 0 10px;
```

Comment:
Just a hint: you can use a shorthand here for `/* top | horizontal | bottom */`

```suggestion
  margin: 30px 10px 0;
```